src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1852,7 +1864,10 @@ private void SetPeriodicEvaluation(WorkspaceModel ws)
           bool forceManualExecutionMode,
           out WorkspaceModel workspace)
         {
-            CustomNodeManager.AddUninitializedCustomNodesInPath(Path.GetDirectoryName(filePath), IsTestMode);
+            if (!String.IsNullOrEmpty(filePath))
```

Comment:
I guess this is good to do, though unnecessary as `ScanNodeHeadersInDirectory` already checks the directory exists and logs something.

Reply:
Ok, will remove